bankroll/converter/modelconverter.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Callable, cast, List, Optional, Sequence, Tuple, Union
+
+try:
+    import pandas as pd
+except ImportError:
+    pass
+
+from bankroll.model import Position, Trade
+
+ConvertibleModel = Union[Position, Trade]
+ColumnFunctions = List[Tuple[str, Callable[[Any], Any]]]
+
+
+def dataframeForModelObjects(
+    items: Sequence[ConvertibleModel]
+) -> Optional[pd.DataFrame]:
```

Comment:
I don't think this should return `Optional`... it should probably throw an exception if `pandas` isn't available, so errors aren't silently missed.
```suggestion
) -> pd.DataFrame:
```

Reply:
✅